docs/reference/migration/migrate_7_0/logging.asciidoc

```diff
@@ -3,6 +3,7 @@
 === Logging changes
 
 [float]
+//tag::notable-breaking-changes[]
```

Comment:
I'd maybe put these changes around JSON log files in the notable highlights but not in the notable breaking changes?